src/pages/video-detail-page/video-detail-page.js

```diff
@@ -8,64 +8,60 @@ import {
 } from './styled';
 import { fetchRelatedVideos } from '../../api';
 import { YOUTUBE_EMBED_URL } from '../../utils';
+import { propTypes } from './prop-types';
 
-class VideoDetailPage extends Component {
-  state = {
-    relatedVideos: { items: [], error: false },
-  };
-
-  componentDidMount() {
-    this.getRelatedVideosList();
-  }
+export const VideoDetailPage = ({ match: { params } }) => {
+  const [relatedVideosList, setRelatedVideosList] = useState([]);
+  const [error, setError] = useState(false);
+  const { videoId, title } = params;
 
-  getRelatedVideosList = async () => {
-    const { videoId } = this.props.match.params;
+  const getRelatedVideosList = async () => {
     const relatedVideos = await fetchRelatedVideos(videoId);
```

Comment:
sam as above. Its better to name the api response as `response` to convey meaningful message